runtime/src/append_vec.rs

```diff
@@ -1230,60 +1225,61 @@ pub mod tests {
 
-        account.set_executable_as_byte(crafted_executable);
+            account.set_executable_as_byte(crafted_executable);
 
-        // reload crafted accounts
-        let accounts = av.accounts(0);
-        let StoredAccountMeta::AppendVec(account) = accounts.first().unwrap();
+            // reload crafted accounts
+            let accounts = av.accounts(0);
+            let StoredAccountMeta::AppendVec(account) = accounts.first().unwrap();
 
-        // upper 7-bits are not 0, so sanitization should fail
-        assert!(!account.sanitize_executable());
+            // upper 7-bits are not 0, so sanitization should fail
+            assert!(!account.sanitize_executable());
 
-        // we can observe crafted value by ref
-        {
-            let executable_bool: &bool = &account.account_meta.executable;
-            // Depending on use, *executable_bool can be truthy or falsy due to direct memory manipulation
-            // assert_eq! thinks *executable_bool is equal to false but the if condition thinks it's not, contradictorily.
-            assert!(!*executable_bool);
-            #[cfg(not(target_arch = "aarch64"))]
+            // we can observe crafted value by ref
             {
-                const FALSE: bool = false; // keep clippy happy
-                if *executable_bool == FALSE {
-                    panic!("This didn't occur if this test passed.");
+                let executable_bool: &bool = &account.account_meta.executable;
+                // Depending on use, *executable_bool can be truthy or falsy due to direct memory manipulation
+                // assert_eq! thinks *executable_bool is equal to false but the if condition thinks it's not, contradictorily.
+                assert!(!*executable_bool);
+                #[cfg(not(target_arch = "aarch64"))]
+                {
+                    const FALSE: bool = false; // keep clippy happy
+                    if *executable_bool == FALSE {
+                        panic!("This didn't occur if this test passed.");
+                    }
                 }
+                assert_eq!(*account.ref_executable_byte(), crafted_executable);
             }
-            assert_eq!(*account.ref_executable_byte(), crafted_executable);
-        }
 
-        // we can NOT observe crafted value by value
-        {
-            let executable_bool: bool = account.executable();
-            assert!(!executable_bool);
-            assert_eq!(account.get_executable_byte(), 0); // Wow, not crafted_executable!
-        }
+            // we can NOT observe crafted value by value
+            {
+                let executable_bool: bool = account.executable();
+                assert!(!executable_bool);
+                assert_eq!(account.get_executable_byte(), 0); // Wow, not crafted_executable!
+            }
 
-        av.flush().unwrap();
-        let accounts_len = av.len();
-        drop(av);
+            av.flush().unwrap();
+            av.len()
+        };
```

Comment:
Btw, is there a convention in Rust to add a tailing comment like the following to make it easier for the reader to understand what this ending bracket is for?

```
        };  // accounts_len
```

Reply:
> I have not seen such convention. Have you?

I learned this convention in Facebook (or Meta).  Probably not used outside of FB.